### DISCUSSION TITLE
Barcamp: Usability Metadateneditor

### DISCUSSION THREAD

**Discussion by SonjaGellert** [kitodo/kitodo-production#5881] [General]
Da das Thema meinen Kolleginnen und mir doch sehr wichtig ist und auch so rege während des Barcamps mit anderen aus der Kitodo-Community besprochen wurde, möchte ich zumindest die Ergebnisse des Flipcarts hier festhalten.

Soweit ich Issues oder ähnliches gefunden habe, habe ich sie dahinter vermerkt, sollte ich jedoch etwas übersehen haben, kann das gerne angemerkt werden.

### Abschrift: Flipcart

**Allgemeines**

- Benutzerdefinierte Einstellungen des ME via Aufgaben & Prozesse #5888 -> MDE Bundle
- Schritt zurück Funktion -> nur schwer umsetzbar, Diskussion beim Community Meeting 20.04.2024

**Strukturbaum**

- Strukturtyp und Titel/Label nicht zusammen sichtbar. #5882 -> MDE Bundle
- mehrere Seiten im Strukturbaum verlinken nicht möglich. #5926 -> MDE Bundle

**Paginierung**

- Nicht ganz robust/konsistent
Es ist aus Anwendersicht, nicht ersichtlich wann die Auswahlfelder sich zurücksetzen und wann nicht, dadurch entstehen häufig Fehler bei der Paginierung
- Seitenübersicht unzureichend
Im Vergleich zu Kitodo 2, ist der Abschnitt der Paginierung unter Kitodo 3 sehr gestaucht und je nach Bildschirmgröße bzw. Auflösung sehr gestaucht
- Paginierung begrenzbar auf Strukturelement (bspw. nur ein Kapitel paginieren)
- Auswahlfeld setzt sich bei Auswahl eines anderen Feldes zurück. #6044

**Metadaten**

- Feld ergänzen über Pluszeichen sollte sofortige Eingabe in Popup ermöglichen #5929 -> MDE Bundle
In der aktuellen Version, wird das Metadatenfeld zwischen den anderen bereits existierenden eingefügt, um nun dieses Feld zu befüllen, muss es gesucht werden. Je nach Anzahl der Metadaten ist das sehr umständlich 
- ~~Fragezeichen in Metadaten nicht möglich~~

**Galerie:**

- Seitenzahl fest sichtbar #5928 -> MDE Bundle
Paginierung bisher nur bei Mouse Over sichtbar, hilfreicher wäre es, wenn es immer bspw. unter dem Bild angezeigt werden würde
- Verknüpfen von Seiten in hierarchischen Strukturen schwierig/unmöglich #5457 – MDE Bundle
Soll ein Scan bspw. 2 Kapiteln zugeordnet werden, ist dies nur einfach möglich, wenn diese Kapitel auf einer hierarchischen Ebene sind. Sobald aber mehrere Hierarchieebenen vorhanden sind, ist es sehr umständlich, diese Verknüpfung herzustellen.

Detailansicht:

- Zoomfaktor merken #5796 -> MDE Bundle
- Einfaches Weiterblättern #5930 -> MDE Bundle
- Bildlaufleiste verbreiten können
- "Festhalten" eines Einzelbildes möglich #5887 – MDE Bundle
Beispiel: Die Seite des Inhaltsverzeichnisses anpinnen, um es bei der Erstellung der Strukturelemente daran zu orientieren
- Parallele Ansicht in Detail und Galerie (auf zwei Bildschirmen) ermöglichen

**Gelöste Probleme/Issues:**
-	Fragezeichen nicht möglich -> hängt mit den Tastenkürzel zusammen! #5819

_Anmerkung von meiner Seite:_
Während dem Gespräch mit den Kollegen, aber auch meiner eigenen Nachrecherche hier im Github, hatte ich teilweise das Gefühl, dass für viele Anwender eine "Sprechstunde" für den Metadataeditor nicht schlecht wäre.
Gerade für Anwender wie mich, ohne oder nur geringem technischen Verständnis ist die Eröffnung einer Diskussion oder eines Issues, doch eine große Hürde.

Update:
Einige Issues wurden aufgrund dieser Diskussion erstellt, die habe ich auch entsprechend verlinkt, soweit es mir möglich war.
Ebenso wurden mit dem Entwicklungsfond 2024 ein Issue-Bundle auf den Weg gegeben #5882

**Comment by subhhwendt** [comment]
Hallo Frau Gellert,
Sie haben recht, an diesem Diskussionsstrang sollte weitergedacht und diskutiert werden! Wir haben uns das Thema im Community Board als nächstes vorgenommen und werden dazu früher oder später sicher auch die ganze Community einladen. An einer anderen - zugegebenermaßen - noch sehr versteckten Stellen habe ich letzte Woche eine ähnliche Liste angelegt: https://github.com/kitodo/kitodo-production/wiki/Barcamp-2023:-Usability-Metadateneditor Diese Liste habe ich jetzt auch um einige Kommentare von Ihnen ergänzt.

**Comment by andre-hohmann** [comment]
Vielen Dank auch von meiner Seite! 
Meine Kolleginnen und Kollegen in der SLUB Dresden fühlen sich bestimmt in einigen Punkten vertreten!

Ich habe noch folgende Anmerkungen: 
1. Einen rudimentären Eintrag im GitHub-Wiki zu den Tastenkombinationen finden Sie unter: [Tastenkombinationen (Shortcut, Tastenkürzel)](https://github.com/kitodo/kitodo-production/wiki/Tastenkombinationen-(Shortcut,-Tastenk%C3%BCrzel)). 
2. Dies ist auch eine Lösungsmöglichkeit für die Eingabe des Fragezeichens: #5819

**Reply by oliver-stoehr** [reply]
@andre-hohmann viele Dank für das Anlegen der Wiki-Seite!

Ich habe die Wiki-Seite um eine ausführliche Beschreibung der Tastaturkürzel-Funktionalität ergänzt.

**Reply by andre-hohmann** [reply]
@oliver-stoehr : Vielen Dank! Jetzt hilft die Seite auch tatsächlich weiter ;-)

**Comment by solth** [comment]
> * Strukturtyp und Titel/Label nicht zusammen sichtbar.

Hierfür habe ich ein Issue erstellt, das die Ergebnisse der verlinkten Diskussion #4925 enthält: #5882

**Reply by SonjaGellert** [reply]
Danke, dass sie ein Issue daraus erstellt haben!

**Comment by solth** [comment]
Github-Issues:
- Hinzufügen von Metadaten samt Werten: https://github.com/kitodo/kitodo-production/issues/5886
- Seite pinnen/in neuem Browser-Fenster öffnen: https://github.com/kitodo/kitodo-production/issues/5887
- Layout für Zugriff von Vorgangsliste speichern: https://github.com/kitodo/kitodo-production/issues/5888

**Comment by SonjaGellert** [comment]
Ich habe die Diskussion nach dem Communitytreffen entsprechend mit neuen Issues geupdated, soweit es mir möglich und nachvollziehbar.

Sollte ich etwas vergessen oder übersehen haben, melden Sie sich gerne hier.
Ebenfalls gilt, sollte noch abseits der genannten Probleme/Wünsche noch weitere Anmerkungen geben, hängen sie sich gerne an die Diskussion mit ran. (werde ich auch noch machen)

Bzw. ich kann gerne alles was im Rahmen des Communitytreffen angesprochen wurde, gerne mit in dieser Diskussion mit anfügen, wenn es gewünscht ist.
